releases/v2.1.0-alpha.20180604.md

```diff
@@ -0,0 +1,214 @@
+- Fix a bug where the sessions endpoint on the Admin UI would return an error when there was an active transaction. [#25249][#25249]
+- Corrected the CockroachDB-specific, currently undocumented conversion from [`INTERVAL`](../v2.1/interval.html) to/from numeric types. [#25257][#25257]
+- Fixed problems with [`IMPORT`](../v2.1/import.html) sometimes failing after node decommissioning. [#25162][#25162]
+- Prevented queries that use placeholders for tuple types from causing a crash. [#25269][#25269]
+- Fixed a rare `segfault` that occurred when reading from an invalid memory location returned from C++. [#25347][#25347]
+- Fixed a bug with `IS DISTINCT FROM` not returning `NULL` values that pass the condition in some cases. [#25336][#25336]
+- Restarting a CockroachDB server on Windows no longer fails due to file system locks in the store directory. [#25267][#25267]
+- Prevented the consistency checker from deadlocking. This would previously manifest itself as a steady number of replicas queued for consistency checking on one or more nodes and would resolve by restarting the affected nodes. [#25456][#25456]
+- Fixed a crash in some cases when using a `GROUP BY` with `HAVING`. [#25574][#25574]
+- Fixed a nil pointer dereference when importing data containing date values. [#25661][#25661]
+- Numeric literal values no longer silently lose information after a certain precision. [#25597][#25597]
+- Prevented spurious `BudgetExceededErrors` for some queries that read a lot of [`JSON`](../v2.1/jsonb.html) data from disk. [#25679][#25679]
+- Fixed query errors in some cases involving a `NULL` constant that is cast to a specific type. [#25735][#25735]
+- Fixed a crash when trying to plan certain `UNION ALL` queries. [#25747][#25747]
+- Fixed a crash caused by inserting data into a table with [computed columns](../v2.1/computed-columns.html) that reference other columns not present in the `INSERT` statement. [#25682][#25682]
+- `EXPLAIN (DISTSQL)` now properly reports that plans containing subqueries cannot be run through the [DistSQL execution engine](../v2.1/architecture/sql-layer.html#distsql). [#25618][#25618]
+- CockroachDB no longer crashes if the control statements `CANCEL`/`PAUSE`/`RESUME` are given values using special PostgreSQL types (e.g., `NAME`). [#25844][#25844]
+- Fixed a panic when using unordered aggregations. [#26042][#26042]
+- Fixed an error caused by [`INET`](../v2.1/inet.html) constants in some rare cases. [#26086][#26086]
+- Fixed an error caused by empty arrays in some cases. [#26090][#26090]
+- Previously, expired compactions could stay in the queue forever. Now, they are removed when they expire. [#26039][#26039]
+- Fixed problems using tables with [foreign key](../v2.1/foreign-key.html) or [interleaved](../v2.1/interleave-in-parent.html) references to other tables when the tables were created in the same transaction. [#25786][#25786]
+
+### Performance Improvements
+
+- Reduced the CPU overhead of idle ranges. [#24956][#24956]
```

Comment:
I'm going to revert this one before the alpha is released.

Reply:
Removed it from the release notes.